XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +333,36 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func selectOptionFromBrowserTabMenu(view: selectMenuOption) {
```

Comment:
```swift
func selectOptionFromBrowserTabMenu(_ view: selectMenuOption) {
```

Swift tip: if you use `_` before the argument name, you don't have to use the named argument when calling:

```swift
navigator.selectOptionFromBrowserTabMenu(.requestDesktop)
```

Second: naming.

Rather than rooting the name in the menu you're taking action from, perhaps you can think of something more high level. e.g.

```swift
navigator.browserAction(.requestDesktop)
```

This way, you can add non menu options e.g. 

```swift
navigator.browserAction(.backButton)
```